app/src/modules/settings/routes/flows/components/logs-drawer.vue

```diff
@@ -4,14 +4,15 @@ import { computed, toRefs, unref } from 'vue';
 import { useI18n } from 'vue-i18n';
 import { getTriggers } from '../triggers';
 import { FlowRaw } from '@directus/types';
-import { useRevision } from '@/composables/use-revision';
+import VDetail from '@/components/v-detail.vue';
 
 const props = defineProps<{
-	revisionId?: number;
 	flow: FlowRaw;
+	revision: any;
+	closeDrawer: () => void;
 }>();
 
-const { revisionId, flow } = toRefs(props);
+const { flow, revision, closeDrawer } = toRefs(props);
```

Comment:
Let’s stick with the `emit` approach

```suggestion
const { flow, revision } = toRefs(props);
```

Reply:
done